ui/src/components/HistoryChart.svelte

```diff
@@ -1,82 +1,87 @@
+  });
 
-    for (let i=0; i < 10; i++){
-        if (scanDur.length < 10) {
-            scanDur.push(maxBarHeight/10)
-        }
+  for (let i = 0; i < 10; i++) {
+    if (scanDur.length < 10) {
+      scanDur.push(maxBarHeight / 10);
     }
+  }
 
-    let score = value.map(i => i.finalEval);
+  let score = value.map((i) => i.finalEval);
 
-    let barColor = score.map(i => {
-    if (i === 'FAIL') return 'red';
-    if (i === 'PASS') return 'green'; 
-    return 'orange';
-    });
-     
-    export let data = {
-            labels: scanDur,
-            datasets: [{
-                data: scanDur,
-                backgroundColor: barColor
-            }]
-        };
-    
-    export let options = {
-            responsive: true,
-            maintainAspectRatio: false,
-            scales:{
-                xAxes: [{
-                    maxBarThickness: 5,
-                    ticks: {
-                        display: false,
-                        beginAtZero: true,
-                        reverse: true
-                    },
-                    gridLines: {
-                        display: false
-                }
-            }],
-                yAxes: [{
-                    ticks: {
-                        display: false,
-                        max: maxBarHeight,
-                        beginAtZero: true,
-                    },
-                    gridLines: {
-                        display: false
-                }
-            }]
-            }, 
-            legend: {
-                display: false
-             },
-        };
+  let barColor = score.map((i) => {
+    if (i === "FAIL") return "red";
+    if (i === "PASS") return "green";
+    return "orange";
+  });
+
+  export let data = {
+    labels: scanDur,
+    datasets: [
+      {
+        data: scanDur,
+        backgroundColor: barColor,
+      },
+    ],
+  };
+
+  export let options = {
+    responsive: true,
+    maintainAspectRatio: false,
+    scales: {
+      xAxes: [
+        {
+          maxBarThickness: 5,
+          ticks: {
+            display: false,
+            beginAtZero: true,
+            reverse: true,
+          },
+          gridLines: {
+            display: false,
+          },
+        },
+      ],
+      yAxes: [
+        {
+          ticks: {
+            display: false,
+            max: maxBarHeight,
+            beginAtZero: true,
+          },
+          gridLines: {
+            display: false,
+          },
+        },
+      ],
+    },
+    legend: {
+      display: false,
+    },
+  };
 
-    let chartRef;
-    onMount(() => {
-        Chart.Bar(chartRef, {
-        options: options,
-        data: data
-        });
+  let chartRef;
+  onMount(() => {
+    Chart.Bar(chartRef, {
+      options: options,
+      data: data,
     });
-    
+  });
 </script>
 
 <div class="text-center">
-<span class="font-sans">HISTORY</span>
+  <span class="font-sans">HISTORY</span>
+  <i class="fas fa-long-arrow-alt-right" />
```

Comment:
we add the whole library just to use 1 icon? find the icon from the same library that we currently use, they have arrow right icon too